.github/workflows/ci.yml

```diff
@@ -1,27 +1,24 @@
 name: CI
 
-on:
-  push:
-    branches:
-      - '*'
-  pull_request:
-    branches:
-      - '*'
+on: [push, pull_request]
 
 jobs:
   build:
     name: MacOS
     runs-on: macOS-latest
     steps:
     - uses: actions/checkout@v2
+      # - name: Setup tmate session
+      #   uses: mxschmitt/action-tmate@v2
     - name: Install Dependencies
-      run: brew uninstall --ignore-dependencies libpq && brew install cmark postgresql || brew update
-      env:
-        HOMEBREW_NO_AUTO_UPDATE: 1
+      run: |
+        brew uninstall --ignore-dependencies libpq
+        brew install cmark postgresql
+        initdb /usr/local/var/postgres
     - name: Start Postgres
-      run: brew services start postgresql || brew update && brew services start postgresql
-    - name: Select Xcode 11.3
-      run: sudo xcode-select -s /Applications/Xcode_11.3.app
+      run: brew services start postgresql
+    - name: Select Xcode 11.4
+      run: sudo xcode-select -s /Applications/Xcode_11.4.app
```

Comment:
🤯 where did you even bump into this trick lol 
is it documented somewhere they have all of these versions installed in that specific path?

Reply:
There's a helpful doc here: https://help.github.com/en/actions/reference/software-installed-on-github-hosted-runners

Specifically if you click through to the Mac environment you get that info here: https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md#xcode

Reply:
Thanks,
Good to know :) Applied this here with a matrix job and works well  https://github.com/CombineCommunity/CombineExt/blob/master/.github/workflows/tests.yml